Dockerfile

```
@@ -1,10 +1,13 @@
-FROM node:18
+# Build stage
+FROM node:18 AS build
```

nit: want to switch to the latest stable version?
```suggestion
FROM node:20 AS build
```

---

Agreed